public/app/plugins/panel/news/NewsPanel.tsx

```diff
@@ -32,7 +32,17 @@ export function NewsPanel(props: NewsPanelProps) {
   }, [getNews]);
 
   if (state.error) {
-    return <div>Error loading RSS feed.</div>;
+    return (
+      <Alert title="Error loading RSS feed">
+        Make sure that the feed URL is correct and that CORS is configured correctly on the server. See{' '}
+        <a
+          style={{ textDecoration: 'underline' }}
+          href="https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/news/"
```

Comment:
Do we have any standard link styles? Also, is this the way we should link to documentation?